content/about/main/index.md

```diff
@@ -7,27 +7,27 @@ number_categories: 3 # set to zero to exclude
 show_intro: true
 intro: |
 
-  I was born in 1977 in [Juan José Paso](https://es.wikipedia.org/wiki/Juan_Jos%C3%A9_Paso_(Buenos_Aires)), a small town in [Argentina](https://es.wikipedia.org/wiki/Argentina). I was six years old when my elementary school bought a computer (one computer for the entire school and the only one in the whole town). The moment I saw it I felt love at first sight.
+  I was born in 1977 in [Juan José Paso](https://es.wikipedia.org/wiki/Juan_Jos%C3%A9_Paso_(Buenos_Aires)), a small town in [Argentina](https://es.wikipedia.org/wiki/Argentina). I was six year old when my elementary school bought a computer (one computer for the entire school and the only one in the whole town). The moment I saw it I felt love at first sight.
   
-  A few years later, we moved to [Ingeniero Luiggi](https://es.wikipedia.org/wiki/Ingeniero_Luiggi) because my family had lost almost everything due to a flood. This new place was close to [Realicó](https://es.wikipedia.org/wiki/Realic%C3%B3) where I could study programming. So between ages 9 and 14 my parents drove 80 kilometers every Saturday to take me to class.  We didn’t have a car, and there was no public transport because it was a rural area, but since my dad repairs vehicles, his clients would lend him their cars to make the trip.
+  A few years later, we moved to [Ingeniero Luiggi](https://es.wikipedia.org/wiki/Ingeniero_Luiggi) because my family had lost almost everything due to a flood. This new place was close to [Realicó](https://es.wikipedia.org/wiki/Realic%C3%B3) where I could study programming. So between age 9 and 14 my parents drove 80 kilometers every Saturday to take me to class.  We didn’t have a car, and there was no public transport because it was a rural area, but since my dad repairs vehicles, his clients would lend him their cars to make the trip.
```

Comment:
Would it make sense for you to create and add a map? :wink:

Reply:
I think I moreover once saw a talk of Laura with a chronological timeline, might be another illustration.

Reply:
(if you had time to create such a timeline, I imagine it can take a lot of time)